server/rules-engine/index.js

```diff
@@ -126,7 +126,7 @@ function RulesEngine () {
   this.Rule = Rule
 
   this.enabledEligibilityRules = function (rules) {
-    return rules.filter(rule => (('types' in rule) && rule.types.includes(enums.rulesTypes.eligibility) && rule.enabled))
+    return rules.filter(rule => (('type' in rule) && rule.type === enums.rulesTypes.eligibility && rule.enabled))
```

Comment:
Some round brackets aren't required in the function.
The `'type' in rule` could be removed completely as `rule.type` would be undefined, so would not match, i.e.

`return rules.filter(rule => rule.type === enums.rulesTypes.eligibility && rule.enabled)`

It raises the question as to why some rules would not have a type, should this be allowed under normal circumstances?

Reply:
Thanks, have updated as per your suggestion. I don't think we understand the business logic enough to know if rules always have a type (Rich is still trying to understand this). Here they are helping to identify where in the application process to apply them. At the moment, eligibility is the only one we check for and all the rest are used at the end.